kodilla-good-paterns/src/main/java/com/kodilla/good/patterns/challenges/NewSale.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.good.patterns.challenges;
+
+import java.time.LocalDate;
+
+public class NewSale implements SaleRepository {
```

Comment:
NewSale? Co to za nazwa dla klasy? To jest repozytorium? To moze SaleRepositoryImpl?